lib/reportData.js

```diff
@@ -225,6 +225,15 @@ ReportData = function (baseURL, mp) {
     };
 
     /**
+     * @instance
+     * Clear the array containing queued objects that will be used for generating urls. This prevents previous previous
```

Comment:
"This prevents _previous previous_ urls from being reported" 
Let's take that whole sentence out.